sys/net/gnrc/network_layer/ipv6/netif/gnrc_ipv6_netif.c

```diff
@@ -877,6 +874,7 @@ void gnrc_ipv6_netif_init_by_dev(void)
 #ifdef MODULE_GNRC_SIXLOWPAN_ND
         if (ipv6_if->flags & GNRC_IPV6_NETIF_FLAGS_SIXLOWPAN) {
 #ifdef MODULE_GNRC_SIXLOWPAN_ND_BORDER_ROUTER
+            bool abr_init = false;
```

Comment:
This looks like a breaking change. `abr_init` will always false in the if statement below. Previously, this could only happen once per loop (starts in line 798 | 95).